code/modules/surgery/organs/ears.dm

```diff
@@ -133,3 +133,22 @@
 	desc = "The robust ears of a bronze golem. "
 	damage_multiplier = 0.1 //STRONK
 	bang_protect = 1 //Fear me weaklings.
+
+/obj/item/organ/ears/cybernetic
+	name = "cybernetic ears"
+	icon_state = "ears-c"
+	desc = "a basic cybernetic designed to mimic the operation of ears."
+	damage_multiplier = 0.9
+	organ_flags = ORGAN_SYNTHETIC
+
+/obj/item/organ/ears/cybernetic/upgraded
+	name = "upgraded cybernetic ears"
+	icon_state = "ears-c-u"
+	desc = "an advanced cybernetic ear, surpassing the performance of organic ears"
+	damage_multiplier = 0.5
+
+/obj/item/organ/ears/cybernetic/emp_act(severity)
+	. = ..()
+	if(. & EMP_PROTECT_SELF)
+		return
+	damage += 40/severity
```

Comment:
am I dumb or does higher severity do LOWER damage.

Prob should just have this as  `(20 * severity)`?

Reply:
Higher severity should do lower damage. Severity one is severe, severity two is mild. I didn't make the convention for this.